laser_slam/src/benchmarker.cpp

```diff
@@ -5,109 +5,219 @@
 #include <iomanip>
 #include <iostream>
 
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
Actually not that easy. Standard C++ is unable to create directories. Let's discuss this tomorrow as well!